packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -51,20 +51,14 @@ class BuildIOSCommand extends FlutterCommand {
 
     printStatus('Building the application for $logTarget.');
 
-    Directory buildDir = new Directory(path.join('build', 'ios_$logTarget'));
```

Comment:
Did this cause problems? It might be nice to have most build artifacts grouped in the `build/` dir.